travis/compile.sh

```diff
@@ -11,6 +11,10 @@ else
 fi
 if [[ "$S390X" == 1 ]]; then
 	S390X_CONFIG="--without-pcre-jit";
+	# Generate resource bundle object for Big Endian arch for the Intl extension
+	icupkg -tb ext/intl/tests/_files/resourcebundle/root.res
+	icupkg -tb ext/intl/tests/_files/resourcebundle/es.res
+	icupkg -tb ext/intl/tests/_files/resourcebundle/res_index.res
```

Comment:
This will still make the tests fail if people run them locally. Instead we should pregenerate and commit the big-endian files (e.g. to `resourcebundle_le` and `resourcebundle_be`) and then pick one or the other based on an endianness check.

Reply:
I thought about this, but didn't know where to make this change, so I just bodged it there for the mean time.
Should this go into autoconf? Could also mean we can drop those files from git tracking

Reply:
Do we also have to cater to word-swapping (see https://stackoverflow.com/questions/2100331/c-macro-definition-to-determine-big-endian-or-little-endian-machine#2103095)?

Reply:
As these files are small, it would be easier to generate them once and commit, than deal with autoconf, icupkg not being available, etc.

Reply:
@cmb69 No, ICU only support LE/BE here. Also: No on principle (also applies to EBCDIC support.)

Reply:
So we need @vibhutisawant or @guirish to generate them for us? Cause I don't think I can locally. But having them in core is probably better than needing to deal with Autoconf...

Reply:
@Girgias Using `icupkg -tb` should work locally as well, I believe.

Reply:
I shall try...